src/Client.php

```diff
@@ -10,7 +10,7 @@
 use Silktide\SemRushApi\Model\Factory\ResultFactory;
 use Silktide\SemRushApi\Model\Request;
 use Silktide\SemRushApi\Model\Result as ApiResult;
-use Guzzle\Http\Client as GuzzleClient;
+use GuzzleHttp\Client as GuzzleClient;
```

Comment:
If I recall correctly, the difference between GuzzleHttp and Guzzle\Http is a breaking change between Guzzle v3 and v4? @andywaite 

GuzzleHttp is what we should be using, you're using Guzzle which is being pulled in as a dev dependancy from php-coveralls. Technically, the code as is isn't working. If you pull down off your non-dev dependancies, it won't work